src/schema/common.cljs

```diff
@@ -8,7 +8,20 @@
 (def ^:private ?customization-color
   [:or :string :keyword])
 
+(def ^:private ?public-key
+  [:re #"^0x04[0-9a-f]{128}$"])
+
+(def ^:private ?general-rpc-call
+  [:cat
+   [:map
+    [:method :string]
+    [:params [:sequential :any]]
+    [:on-success fn?]
+    [:on-error fn?]]])
+
 (defn register-schemas
   []
   (registry/register ::theme ?theme)
-  (registry/register ::customization-color ?customization-color))
+  (registry/register ::customization-color ?customization-color)
+  (registry/register ::public-key ?public-key)
+  (registry/register ::general-rpc-call ?general-rpc-call))
```

Comment:
Do you think we'll have a global & specific schema for `:json-rpc/call`? I currently only see utility in a generic global one, and then the developer can override specific parts using `[:merge ...]`.

I would say the word `general` can be safely removed.

Side note: `:json-rpc/call` wants a sequence of maps, but this is used only in a few places. Since we can conveniently dispatch multiple effects using `:fx`, this part of the API could be simplified, which would help also simplify the schema.